resolwe_bio/descriptors/default_clinical.yml

```diff
@@ -0,0 +1,1009 @@
+- slug: default-clinical
+  name: Default clinical annotation
+  description: Default clinical meta data.
+  version: 1.0.0
+  schema:
+    - name: general
+      label: General
+      group:
+        - name: species
+          label: Species
+          type: basic:string
+          default: Homo sapiens
+          required: false
+        - name: annotator
+          label: Annotator
+          type: basic:string
+          required: false
+    - name: subject_information
+      label: Subject information
+      group:
+        - name: sample
+          label: Sample
+          type: basic:string
+          required: false
+        - name: subject_id
+          label: Subject ID
+          type: basic:string
+          required: false
+        - name: batch
+          label: Batch
+          type: basic:integer
+          required: false
+        - name: group
+          label: Group
+          type: basic:integer
+          required: false
+        - name: gender
+          label: Gender
+          type: basic:string
+          required: false
+          choices:
+            - label: M
+              value: m
+            - label: F
+              value: f
+        - name: race
+          label: Race
+          type: basic:string
+          required: false
+          choices:
+            - label: White
+              value: white
+            - label: Non-White
+              value: non-white
+            - label: Asian
+              value: asian
+            - label: Non-Asian
+              value: non-asian
+            - label: Black
+              value: black
+            - label: Non-Black
+              value: non-black
+        - name: ethnicity
+          label: Ethnicity
+          type: basic:string
+          required: false
+          choices:
```

Comment:
Values in this section have whitespace in them, whereas other names/values do not.

Reply:
It's whitespace in choices values but _ in names according to the spec.